music/src/core/player.ts

```diff
@@ -212,25 +221,36 @@ export abstract class BasePlayer {
   }
 
   /**
-   * Pause playing the currently playing sequence right away. Call unpause()
+   * Pause playing the currently playing sequence right away. Call resume()
    * to resume.
+   * @throws {Error} If the player is stopped.
    */
   pause() {
+    if (!this.isPlaying()) {
+      throw new Error('Cannot pause playback while the player is stopped.');
+    }
     Tone.Transport.pause();
   }
 
   /**
-   * Pause playing the currently playing sequence right away. Call resume()
-   * to resume playing the sequence.
+   * Resume playing the sequence after pause().
+   * @throws {Error} If the player is not paused.
    */
   resume() {
+    if (this.getPlayState() !== 'paused') {
+      throw new Error(`Cannot resume playback while "${this.getPlayState()}".`);
+    }
     Tone.Transport.start();
   }
 
   /**
    * Seek to a number of seconds in the NoteSequence.
+   * @throws {Error} If the player is stopped.
    */
   seekTo(seconds: number) {
+    if (!this.isPlaying()) {
```

Comment:
Not sure about this one either. You can definitely seek while the player is paused or stopped. I would remove this

Reply:
Same as above: `isPlaying()` returns `true` when the player is paused. And when it's stopped, you can't `resume()` it anymore; you can only call `start()` which will start playing from time 0. Am I correct?

Reply:
In any case, we don't want to allow doing anything with the `Transport` when the player is stopped, because it might affect some other player.